### PR TITLE
RNs 4 6 18

### PR DESCRIPTION
**Pull request by stevsmit**
Adds release notes 4.6.18 for 02-22-01. 
Preview will generate.

### CODE REVIEW

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -2586,3 +2586,20 @@ link:https://access.redhat.com/solutions/5798061[{product-title} 4.6.17 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.6 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+[id="ocp-4-6-18"]
+=== RHBA-2021:0510 - {product-title} 4.6.18 bug fix update
+
+Issued: 2021-02-22
+
+{product-title} release 4.6.18 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2020:0510[RHBA-2020:0510] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2020:0511[RHBA-2020:0511] advisory.
```
Comment (codyhoag) [inline release_notes/ocp-4-6-release-notes.adoc:2595]:
```suggestion
{product-title} release 4.6.18 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2021:0510[RHBA-2021:0510] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2021:0511[RHBA-2021:0511] advisory.
```

**Review by codyhoag — COMMENTED** [review]
Just some year changes needed. Looks great otherwise! Nice work!

**Review by kalexand-rh — COMMENTED** [review]
LGTM

**Review by adellape — APPROVED** [review]
:+1:

**Pull request merged**